packages/jsii-pacmak/test/expected.jsii-calc-lib/golang/submodule.go

```diff
@@ -0,0 +1,53 @@
+
+// enum EnumFromScopedModule
+type IDoublable interface {
+    getdoublevalue() number
+    setdoublevalue()
```

Comment:
This seems to have overriden the changes in https://github.com/aws/jsii/commit/20ab90cb60df3fd7fadd18144f71c1f280928f15